packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -16,11 +16,11 @@ describe('Collection aggregations tab', function () {
     browser = compass.browser;
 
     await browser.connectWithConnectionString('mongodb://localhost:27018/test');
-
-    await browser.navigateToCollectionTab('test', 'numbers', 'Aggregations');
   });
 
   beforeEach(async function () {
+    // Some tests navigate away from the numbers collection aggregations tab
+    await browser.navigateToCollectionTab('test', 'numbers', 'Aggregations');
```

Comment:
@lerouxb does this makes sense to you? I can also do this explicitly at the end of every test that navigates away from the aggregation tab. What do you think is better?

Reply:
I've been in two minds about that myself. In the past I tried to restore the state at the end of the test that navigates away, but inevitably that test fails and then the next tests after them are all guaranteed to fail. So I think this is probably safest.

I'm not 100% sure that browsing to a tab that you're already on is well tested yet, but I suppose we'll find out and then just have to fix it if there are problems.

Reply:
I'll keep it then, I did a few runs locally and this method seems to handle stuff well (CI seems to agree too)